static/css/cdr-homepage.css

```diff
@@ -246,10 +250,275 @@ footer .footer-menu {
 }
 footer a {
   color: white;
-  font-size: .8rem;
+  font-size: 16px;
 }
 footer a:hover, footer a:active {
   color: #4B9CD3;
 }
 
+/**
+*
+* Search pages
+*
+**/
+.search-query-text {
```

Comment:
I'm a little unclear on what's happening with the various CSS files. It seems like historically this file (`cdr-homepage.css`) just had stuff related to the front page and a bit of navigation in it, but it now has search page styles (which appear to be duplicated from dcr.scss). I think the compiled css is supposed to be going into `cdr_access.css`. 

I also wonder if it'd make sense to divide search results css into its own sass file versus all going into dcr.scss, but i'm not as familiar with sass practices.